gerrychain/updaters/flows.py

```diff
@@ -2,6 +2,16 @@
 import functools
 
 
+@functools.lru_cache(maxsize=2)
+def neighbor_flips(partition):
+    """The neighbors of the flips in a partition"""
+    return {
+        tuple(sorted((node, neighbor)))
```

Comment:
I understand that the reason vertices are sorted is to prevent duplicate edges – is there a better way to do this? There may not be – and it's probable that it wouldn't make much of a difference anyway – but it's something to think about.

Reply:
I don't think there is a faster way to remove duplicate edges (I tried some other methods, but this seemed to work the best and guarantee equivalence with the previous version of the code).